packages/code-review-gpt/src/common/model/AIModel.ts

```diff
@@ -17,19 +17,25 @@ interface IAIModel {
 const defaultRetryCount = 3;
 
 class AIModel {
-  private model: OpenAIChat;
+  private model: ChatOpenAI;
   private retryCount: number;
 
   constructor(options: IAIModel) {
     switch (options.provider) {
       case "openai":
-        this.model = new OpenAIChat({
+        this.model = new ChatOpenAI({
           openAIApiKey: options.apiKey,
           modelName: options.modelName,
           temperature: options.temperature,
           configuration: { organization: options.organization },
         });
         break;
+      case "azureai":
+        this.model = new AzureChatOpenAI({
```

Comment:
The settings are not set here, as you can use the EnvVars and the package will pick those up by default. 

e.g:

OPENAI_API_KEY=
AZURE_OPENAI_API_INSTANCE_NAME="oai-4turbo"
AZURE_OPENAI_API_DEPLOYMENT_NAME="gpt-4-turbo"
AZURE_OPENAI_API_KEY=xxxx
AZURE_OPENAI_API_VERSION="2024-08-01-preview"